test.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
-set -euo pipefail
+set -exuo pipefail
```

Comment:
Added `x` here since both `test.sh` and `test-prow-e2e.sh` are executed on the Prow CI env. and it helps to see the immediate commands prior to their execution.